packages/klighd-core/src/depth-map.ts

```diff
@@ -369,6 +386,7 @@ export class DepthMap {
      */
     computeDetailLevel(region: Region, viewport: Viewport, relativeThreshold: number, scaleThreshold: number): DetailLevel {
         if (!this.isInBounds(region, viewport)) {
+            console.log(region.boundingRectangle.id + " out of bounds")
```

Comment:
debug log or message important for the user?

Reply:
I think this was for debugging and will be removed